src/shared/dropdown-list/dropdown-list.styles.tsx

```diff
@@ -87,6 +98,19 @@ export const ListItem = styled.li<ListItemProps>`
     }}
 `;
 
+const _getMinHeight = (props: ListItemSelectorProps) => {
```

Comment:
do update the other instances, the prefix is not needed

```suggestion
const getMinHeight = (props: ListItemSelectorProps) => {
```